internal/handler/bothandler/send.go

```diff
@@ -0,0 +1,40 @@
+package bothandler
+
+import (
+	"backend/internal/dataaccess/chat"
+	"backend/internal/database"
+	"backend/internal/params/tgmessageparams"
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+)
+
+func SendMessage(c echo.Context) error {
+	chatIDStr := c.Param("chat_id")
+	chatID, err := strconv.Atoi(chatIDStr)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, "Invalid chat_id "+err.Error())
+	}
+	r := new(tgmessageparams.MessageParams)
+	if err := c.Bind(&r); err != nil {
+		return c.JSON(http.StatusBadRequest, err)
+	}
+	if err := c.Validate(r); err != nil {
+		return c.JSON(http.StatusBadRequest, err)
+	}
+
+	db := database.GetDb()
+
+	chat, err := chat.Read(db, uint(chatID))
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, err)
+	}
+
+	_, err = SendTelegramMessage(int64(chat.TelegramChatId), nil, r.Message)
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, err)
+	}
```

Comment:
After sending the tg message, it would be good to save the `*tgbotapi.Message` to db with `saveTgMessageToDB` from `internal/handler/tgmessagehandler/util.go`. Feel free to make the util function public or add it to `util.go`

If telegram updates the backend bot automatically, where `bot.GetUpdatesChan()` will receive the update, then you can just ignore the above suggestion.

Reply:
Ah yes right, I forgot about this